modules/plotting/properties/dataframeproperty.cpp

```diff
@@ -39,8 +39,7 @@ DataFrameColumnProperty::DataFrameColumnProperty(std::string identifier, std::st
                                                  bool allowNone, size_t firstIndex)
     : OptionPropertyInt(identifier, displayName)
     , dataframe_(nullptr)
-    , allowNone_(allowNone)
-    , firstIndex_(firstIndex) {
```

Comment:
what about initializing `firstIndex_`? This PR does not include changes in dataframeproperty.h.